enderio-machines/src/main/java/com/enderio/machines/common/energy/EnergyTransferMode.java

```diff
@@ -0,0 +1,7 @@
+package com.enderio.machines.common.energy;
+
+public enum EnergyTransferMode {
+    None,
+    Insert,
+    Extract
```

Comment:
No both?

Reply:
Not taking into account individual sides, it'd only receive or send power at any given time. This is an old enum that's planned to be replaced in my energy rework PR.